config/google_config.yaml

```diff
@@ -0,0 +1,3 @@
+---
+  spreadsheet_id_1: 1bFSDYFT5fBpDQTvAeqw4j7QhYXTnFmDuGCLGDwx-wYk
```

Comment:
As mentioned, these might be better with descriptive names which help the maintainer understand the contents.